src/consumer_api_service.ts

```diff
@@ -301,6 +319,93 @@ export class ConsumerApiService implements IConsumerApi {
     });
   }
 
+  // ManualTasks
+  public async getManualTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<ManualTaskList> {
+    const suspendedFlowNodes: Array<Runtime.Types.FlowNodeInstance> =
+      await this.flowNodeInstanceService.querySuspendedByProcessModel(processModelId);
+
+    const manualTaskList: ManualTaskList = await this.manualTaskConverter.convertManualTasks(identity, suspendedFlowNodes);
+
+    return manualTaskList;
+  }
+
+  public async getManualTasksForCorrelation(identity: IIdentity, correlationId: string): Promise<ManualTaskList> {
+    const suspendedFlowNodes: Array<Runtime.Types.FlowNodeInstance> =
+      await this.flowNodeInstanceService.querySuspendedByCorrelation(correlationId);
+
+    const manualTaskList: ManualTaskList = await this.manualTaskConverter.convertManualTasks(identity, suspendedFlowNodes);
+
+    return manualTaskList;
+  }
+
+  public async getManualTasksForProcessModelInCorrelation(identity: IIdentity,
+                                                          processModelId: string,
+                                                          correlationId: string): Promise<ManualTaskList> {
+
+    const suspendedFlowNodes: Array<Runtime.Types.FlowNodeInstance> =
+      await this.flowNodeInstanceService.querySuspendedByCorrelation(correlationId);
+
+    const suspendedProcessModelFlowNodes: Array<Runtime.Types.FlowNodeInstance> =
+      suspendedFlowNodes.filter((flowNodeInstance: Runtime.Types.FlowNodeInstance) => {
+        return flowNodeInstance.tokens[0].processModelId === processModelId;
+      });
+
+    const manualTaskList: ManualTaskList =
+      await this.manualTaskConverter.convertManualTasks(identity, suspendedProcessModelFlowNodes);
+
+    return manualTaskList;
+  }
+
+  public async finishManualTask(identity: IIdentity,
+                                processModelId: string,
+                                correlationId: string,
+                                manualTaskId: string): Promise<void> {
+
+    const manualTasks: ManualTaskList = await this.getManualTasksForProcessModelInCorrelation(identity, processModelId, correlationId);
+
+    const manualTask: ManualTask = manualTasks.manualTasks.find((task: ManualTask) => {
+      return task.id === manualTaskId;
+    });
+
+    if (manualTask === undefined) {
+      const errorMessage: string = `Process model '${processModelId}' in correlation '${correlationId}'
+       does not have a manual task '${manualTaskId}'`;
+      throw new EssentialProjectErrors.NotFoundError(errorMessage);
+    }
+
+    return new Promise<void>((resolve: Function, reject: Function): void => {
+
+      const manualTaskFinishedEvent: string = Messages.EventAggregatorSettings.routePaths.manualTaskFinished
```

Comment:
```suggestion
      const manualTaskFinishedEvent: string = Messages.EventAggregatorSettings
            .routePaths.manualTaskFinished
```

So wird einfach klar, was da passiert.